services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfeLinkBolt.java

```diff
@@ -106,9 +107,9 @@
     private static final String STREAM_ID_CTRL = "ctrl";
     @VisibleForTesting
     static final String STATE_ID_DISCOVERY = "discovery-manager";
-    static final String TOPO_ENG_STREAM = "topo.eng";
     static final String SPEAKER_DISCO_STREAM = "speaker.disco";
     static final String SPEAKER_STREAM = "speaker";
+    static final String NETWORK_TOPOLOGY_CHANGE_STREAM = "network-topology-change-stream";
```

Comment:
long names ruins topology visualization in storm UI.

Remove at least "-stream" suffix from.

(minor)